functions/enqueues.php

```diff
@@ -2,16 +2,39 @@
 /**
  * Enqueue scripts and styles.
  */
+function bootscout_theme_preconnect() {
+    echo '<link rel="preconnect" href="https://fonts.googleapis.com">' . PHP_EOL;
+    echo '<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>' . PHP_EOL;
+}
+
 function bootscout_theme_scripts() {
 	// Styles
-
-	wp_register_style('BootstrapIcons', 'https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.0/font/bootstrap-icons.css', false, '1.7.0', null);
+    wp_register_style(
+        'Nunito+Sans',
+        'https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,wght@0,400;0,600;0,700;0,900;1,400;1,600;1,700;1,900&display=swap',
```

Comment:
Loading styles: 400 (normal), 600 (used in navigation), 700 (bold), 900 (used in h1) and italics versions of all.